app/Http/Controllers/ServerController.php

```diff
@@ -38,6 +33,16 @@ public function create()
             return redirect()->route('servers.index')->with('error', "You've already reached your server limit!");
         }
 
+        //Required Verification for creating an server
+        if(Configuration::getValueByKey('VERIFIED_EMAIL_TO_MAKE_SERVER') === 'true' && !Auth::user()->hasVerifiedEmail()) {
```

Comment:
You can add a default value to 'getValueByKey' by using the second param ('KEY' , 'DEFAULT_VALUE') that way if the key is somehow missing this would still work and not trow errors or unexpected behaviour :)

Reply:
Done in new PR (#95) (sadly I was not able to commit into this one)